metrics/storage/README.md

```diff
@@ -0,0 +1,10 @@
+# Storage metrics tests
+
+This section includes metrics tests focused on measure storage bandwith under different
+conditions/configuration where could be executed different I/O operation such as: read,
+write, random read etc.
+However the use of these should be complemented with more tools in order to find the hot spot.
+
+- `fio_job.sh`: measures the bandwidth measured by a specif I/O operation (read, write,
```

Comment:
s/specif/specific
Also, Im confused by the double use of "measure". Could you replace the second "measure" with a word that better describes what the I/O operations are doing?

Reply:
I am going to rephrase that :)